legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/dbExtension.pure

```diff
@@ -71,6 +71,12 @@ Class meta::relational::functions::sqlQueryToString::DbConfig
      $this.dbExtension.semiStructuredElementProcessor->toOne()->eval($s, $sgc);
    }: String[1];
 
+   semiStructuredPathRegex()
+   {
+     assert($this.dbExtension.semiStructuredPathRegex->isNotEmpty(), '[unsupported-api] Semi structured path navigation not supported for Database Type: ' + $this.dbType->toString());
+     $this.dbExtension.semiStructuredPathRegex->toOne();
+   }: String[1];
+
```

Comment:
Regex should not be db specific. Users should write the navigation in the form platform expects and platform then cross compiles it to db specific navigation. This enables reuse of same store/mapping across different database types.

Reply:
The path is actually the JSON path spec, so I have updated to use the same and cross compile into the SingleStore navigation as suggested.

Reply:
The path is actually the JSON path spec, so I have updated to use the same and cross compile into the SingleStore navigation as suggested.

Reply:
The path is actually the JSON path spec, so I have updated to use the same and cross compile into the SingleStore navigation as suggested.